lib/winnow/model.rb

```diff
@@ -53,9 +56,14 @@ def search(all_params)
       def accepted_name?(name)
         column_names.include?(name.to_s) ||
           contains_scopes.include?(name.to_s) ||
+          starts_with_scopes.include?(name.to_s) ||
           respond_to?(name)
       end
 
+      def starts_with_scopes
+        @starts_with_scopes ||= column_names.map { |name| "#{name}_starts_with" }.flatten
+      end
+
```

Comment:
very minor nitpick, but it would be nice if `starts_with_scopes` & `contains_scopes` were defined in the order they are called.